specifyweb/frontend/js_src/lib/components/Formatters/Definitions.tsx

```diff
@@ -152,7 +152,7 @@ function ConditionalFormatter({
             </div>
             <span className="-ml-2" />
             {trimmedFieldsLength > 0 && isExpanded ? (
-              <Button.Danger onClick={handleDelete}>
+              <Button.Danger disabled={isReadOnly} onClick={handleDelete}>
```

Comment:
instead of disabling, can we hide it?
pros:
- hiding it makes UI simpler
- we usually do disabling for things that could get enabled (i.e disabled because it's currently loading or because some field is empty). in this case, it's about user permissions, so that is more persistent (at very least won't change during the user's session), so hiding the element is a better option

Reply:
yes, you're totally right